Assets/AppCenter/Editor/CustomDropDownPropertyDrawer.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+[CustomPropertyDrawer(typeof(CustomDropDownPropertyAttribute))]
+public class CustomDropDownPropertyDrawer : PropertyDrawer
+{
+    bool initialized = false;
```

Comment:
Those private fields should use `_` prefix.

Reply:
Fixed